commands/update.js

```diff
@@ -0,0 +1,167 @@
+ */
+export const cleanVersion = (originVersion = "") => originVersion.replace(/[^\d.]/g, "");
+
+/**
+ * @summary Get plugin chooses list should to update
+ * @param {Array<string>} plugins - Plugins need to update
+ * @returns {Array<string>} - return array of plugin chooses
+ */
+export const getShouldUpdatePlugins = (plugins) => plugins.map(({ name, version, remoteVersion }) => ({
+  name: `${name}\t ${version} -> ${remoteVersion}`
+}));
+
+/**
+ * @summary Update the package json data
+ * @param {Object} answers - Answer object
+ * @returns {Object} - return updated package json data
+ */
+export const updatePackageJson = ({ answers, localPacakgeJson, outdatedPackages }) => {
+  const updatedPackageJson = { ...localPacakgeJson };
+
+  if (answers && answers.plugins) {
+    const pendingPlugins = answers.plugins.map((plugin) => plugin.split("\t")[0]);
+    outdatedPackages.forEach(({ name, remoteVersion }) => {
+      if (pendingPlugins.includes(name)) {
+        updatedPackageJson.dependencies[name] = remoteVersion;
+      }
+    });
+  } else {
+    outdatedPackages.forEach(({ name, remoteVersion }) => {
+      updatedPackageJson.dependencies[name] = remoteVersion;
+    });
+  }
+  return updatedPackageJson;
+};
+
+/**
+ * @summary allow user can update plugins
+ * @param {Object} options - Any options for project creation
+ * @returns {boolean} - return true if successful
+ */
+export default async function update(options) {
+  if (!await pathExists(PACKAGE_JSON_PATH)) {
+    Logger.error("The current directory does not contain package.json");
+    return false;
+  }
+
+  Logger.info("Getting the local dependencies");
+  const packageJsonData = await getLocalPackageJson();
+  const { dependencies } = packageJsonData;
+  if (!dependencies) {
+    Logger.error("Dependencies object not found");
+    return false;
+  }
+
+  const packagePrefix = "@reactioncommerce/";
+  const pluginPackages = Object.entries(dependencies)
+    .map(([name, version]) => ({ name, version }))
+    .filter(({ name }) => name.startsWith(packagePrefix));
+
+  if (pluginPackages.length === 0) {
+    Logger.info("Plugins not found.");
+    return true;
+  }
+
+  Logger.info("Getting the remote dependencies");
+  const remoteDependencies = await getRemoteDependencies();
+
+  const outdatedPackages = pluginPackages
+    .map(({ name, version }) => ({
+      name,
+      version,
+      remoteVersion: remoteDependencies[name],
+      outdated: compareVerions.compare(cleanVersion(remoteDependencies[name]), cleanVersion(version), ">")
+    }))
+    .filter(({ outdated }) => outdated);
+
+  if (outdatedPackages.length === 0) {
+    Logger.info("All plugins are up to date.");
+    return true;
+  }
+
+  Logger.info(`Found ${pluginPackages.length} plugins. ${outdatedPackages.length} outdated plugins.`);
+
+
+  const answers = options.all ? null : await inquirer.prompt([
+    {
+      type: "list",
+      name: "kind",
+      message: "Do want to update your outdate plugins?",
```

Comment:
Outdate is a verb, so `outdated` is the correct spelling here.
```suggestion
      message: "Do want to update your outdated plugins?",
```